modules/farm_fd2_school/src/entrypoints/FD3_Application/App.vue

```diff
@@ -11,24 +11,19 @@
       v-model:date="date"
     />
     <br />
-    <label
-      for="harvest-crop"
-      class="label-margin"
-    >
-      Crop:
-    </label>
-    <select
-      id="harvest-crop"
-      v-model="crop"
-    >
-      <option
-        v-for="crop in cropList"
-        v-bind:key="crop.id"
-        v-bind:value="crop"
-      >
-        {{ crop.attributes.name }}
-      </option>
-    </select>
+
+    <CropSelector
+      v-bind:required="true"
+      v-bind:showValidityStyling="validity.showStyling"
+      v-model:selected="form.crop"
+      v-on:valid="
+        (valid) => {
+          validity.crop = valid;
+        }
+      "
+      v-on:ready="createdCount++"
```

Comment:
You are not using `createdCount` in any important way.  It is in the example code because full FarmData2 pages will use it. But it is not something you are using, so this could be omitted to simplify your code.